gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_0_0/schema.yml

```diff
@@ -9,16 +9,13 @@ databaseChangeLog:
         - createTable:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.